src/utils.jl

```diff
@@ -10,9 +10,10 @@ triggered to regenerate the documents, subsequently the LiveServer will render t
 in `docs/build`.
 """
```

Comment:
Please fix the function signature & add a short note about folder name being the name of the folder if not docs

Reply:
My bad, forgot the signatures.